deploy/kiali/kiali_cr.yaml

```diff
@@ -697,6 +697,22 @@ spec:
 #    ---
 #    ui_defaults:
 #
+# Default presets for Graph Find/Hide
+#      ---
+#      graph:
+#        find_options:
+#          - description: "Find: slow edges (> 1s)"
+#            expression: "rt > 1000"
+#          - description: "Find: unhealthy nodes"
+#            expression:  "! healthy"
+#          - description: "Find: unknown nodes"
+#            Expression:  "name = unknown"
+#        hide_options:
+#          - description: "Hide: healthy nodes"
+#            expression: "healthy"
+#          - description: "Hide: unknown nodes"
+#            expression:  "name = unknown"
```

Comment:
Follow the conventions everywhere else in the operator - we do not indent lists twice. So this should be:
```suggestion
#        find_options:
#        - description: "Find: slow edges (> 1s)"
#          expression: "rt > 1000"
#        - description: "Find: unhealthy nodes"
#          expression:  "! healthy"
#        - description: "Find: unknown nodes"
#          Expression:  "name = unknown"
#        hide_options:
#        - description: "Hide: healthy nodes"
#          expression: "healthy"
#        - description: "Hide: unknown nodes"
#          expression:  "name = unknown"
```